src/main/java/com/psychology/product/block/test/entities/Answer.java

```diff
@@ -0,0 +1,43 @@
+package com.psychology.product.block.test.entities;
```

Comment:
Change package structure
something like poduct.test. and each entity should stay in own package